hardhat/hardhat.config.js

```diff
@@ -0,0 +1,34 @@
+require('@nomiclabs/hardhat-waffle')
+
+// Keys
+/* eslint-disable-next-line */
+const { ALCHEMY_API_KEY, ARBITRUM_TESTNET_PRIVATE_KEY } = require('./keys')
```

Comment:
@noahdahlman I think my local stat ie mess up, but this was a local file that git ignored which basically just exported out the .env values as constants that could be reused elsewhere.  This was just a quick workaround for me to not commit the values into git.